cli/tools/test/mod.rs

```diff
@@ -399,6 +399,33 @@ impl TestFailure {
     }
   }
 
+  pub fn error_location(&self) -> Option<TestLocation> {
+    match self {
+      // TODO: What happens if not enough call stack frames were recorded?
+      // TODO: Could anything else trigger the ext:cli/40_test.js file? Maybe we should also test for the method name
```

Comment:
Yes, it's quite easy to for something else to trigger a `ext:` file. Like an illegal call to `Deno.test` from within a test.

![image](https://github.com/user-attachments/assets/19660e6d-0ca7-402f-a31e-cee14ff6f618)

Reply:
I did find a case where it does fail catastrophically

![image](https://github.com/user-attachments/assets/e00d5984-22f9-4d8c-a8b4-addb454f7c5f)

So we do need a more specific test.

Code is
```
import { A } from "./foo.ts";

// hmm

Deno.test("database operations", (t) => {
  Deno.inspect(new A());
});
```
and that, in another ifle
```
export class A {
 [Symbol.for("Deno.customInspect")]() {
    throw new Error("over here!");
 }
}
```

Reply:
Finding the first entry in the stack that points at the current test file could be done. To do so, we would repeatedly call `uri_parse_unencoded(v.file_name)` until we find a URI that matches.

However, this wouldn't necessarily point at the correct line in the stack. Instead it'd point at whatever happens to come first in the current file.
![image](https://github.com/user-attachments/assets/ce262194-f00f-4888-a653-324473a91777)